GrailDiary/GrailDiary/Views/POITableViewCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  POITableViewCell.swift
+//  GrailDiary
+//
+//  Created by Mike Nichols on 5/14/20.
+//  Copyright © 2020 Mike Nichols. All rights reserved.
+//
+
+import UIKit
+
+class POITableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var countryLabel: UILabel!
+    @IBOutlet weak var cluesNumberLabel: UILabel!
+
+    override func awakeFromNib() {
```

Comment:
Feel free to remove this. It won't be used.

Reply:
you mean the awakefromnib?